src/ol/layers/TrackerLayer.js

```diff
@@ -25,22 +25,43 @@ class TrackerLayer extends mixin(Layer) {
       ...options,
     });
 
+    let oldCenter = null;
+    let oldZoom = null;
+
     this.olLayer =
       options.olLayer ||
       new Group({
         layers: [
           new OLLayer({
             source: new Source({}),
             render: (frameState) => {
-              if (this.tracker && this.tracker.canvas) {
+              if (!this.tracker || !this.tracker.canvas) {
+                return null;
+              }
+              const { zoom, center } = frameState.viewState;
+              oldZoom = oldZoom || zoom;
+              oldCenter = oldCenter || center;
+
+              if (zoom !== oldZoom) {
```

Comment:
I think you can use this rendeState like in MapboxLayer https://github.com/geops/mobility-toolbox-js/blob/master/src/ol/layers/MapboxLayer.js#L70

maybe it's more clean.

Reply:
I've made the changes.

Reply:
💋  Sorry, this wasn't my strategy.